cmd/vicadmin/vicadm.go

```diff
@@ -529,9 +547,22 @@ func (s *server) listenPort() int {
 	return s.l.Addr().(*net.TCPAddr).Port
 }
 
+func basicHTTPAuthenticator(handler http.HandlerFunc) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		user, password, ok := r.BasicAuth()
+		if !ok || (user != "root" || password != "thisisinsecure") {
```

Comment:
I don't believe HTTP auth will ever be used in a prod environment -- it's supposed to just exist for testing once we have an authentication mechanism for all of VIC to hook into.